packages/explat-react-client/src/index.ts

```diff
@@ -1,2 +1,50 @@
-// Stub index
-export default null;
+import { useState, useEffect } from 'react'
+
+import type { ExPlatClient, ExperimentAssignment } from '@automattic/explat-client'
+
+interface ExPlatReactClient {
+    /**
+     * An ExPlat useExperiment hook.
+     *
+     * NOTE: Doesn't obey ExperimentAssignment TTL in order to keep stable UX.
+     *
+     * @returns [isExperimentAssignmentLoading, ExperimentAssignment | null]
+     */
+    useExperiment: (experimentName: string) => [boolean, ExperimentAssignment | null]
+}
+
+export default function createExPlatReactClient(ExPlatClient: ExPlatClient): ExPlatReactClient {
```

Comment:
Looks like there are a few styling issues in general, but the name of the variable instance should definitely not be the same as the name of the interface. :slightly_smiling_face: